src/app/globals.css

```diff
@@ -75,6 +75,13 @@ li:active,
     rgba(99, 4, 54, 1) 90%
   ) !important;
 }
+@media (max-width: 768px) {
+  .ant-menu-submenu-popup {
+    left: 0 !important;
```

Comment:
See, that breaks if screen is wider than submenu+menu width.

![image](https://github.com/user-attachments/assets/f0bf10f2-8aea-4fb0-975a-e66e1ded4c59)